src/nft/components/collection/ActivityCells.tsx

```diff
@@ -110,7 +110,14 @@ export const AddressCell = ({ address, desktopLBreakpoint }: AddressCellProps) =
       display={{ sm: 'none', xl: desktopLBreakpoint ? 'none' : 'flex', xxl: 'flex' }}
       className={styles.addressCell}
     >
-      <Box>{address ? shortenAddress(address, 2, 4) : '-'}</Box>
+      <a
+        target="_blank"
+        rel="noreferrer"
+        href={`https://etherscan.io/address/${address}`}
+        style={{ textDecoration: 'none' }}
```

Comment:
Shouldn't you use `ExternalLink` component here? and for the etherscan link there seems to be `getExplorerLink` available which seems the right one.

Reply:
thank you! great catch appreciate it.